integration_tests/pyproject.toml

```
@@ -17,9 +17,10 @@ protobuf = "^3.13.0"
 grpcio = "^1.33.2"
 PyYAML = "^5.3.1"
 python-dateutil = "^2.8.1"
-pystarport = { git = "https://github.com/crypto-com/pystarport.git", branch = "main" }
 web3 = "^5.20.1"
 eth-bloom = "^1.0.4"
+python-dotenv = "^0.19.2"
+pystarport = {git = "https://github.com/crypto-com/pystarport.git", rev = "main"}
```

```suggestion
pystarport = "^0.2.3"
```
`0.2.3` is released on pypi now,